share/wake/lib/core/string.wake

```diff
@@ -59,6 +59,16 @@ export def explode (string: String): List String =
 
     p string
 
+# implode: join a List String into a String with an optional
+# combining string between each one.
+#
+# ```
+# implode ", " ("hello", "world",) = "hello, world"
+# implode "" ("a", "b", "c",) = "abc"
+# ```
+def implode (combiner: String) (strings: List String) =
```

Comment:
looks like implode leaves combiner on the end of the list.

I think this is necessary since it becomes the final terminal but at the very least we shouldn't change it without a strong justification since it would invalidate everyone's databases